zerver/lib/actions.py

```diff
@@ -2334,6 +2334,9 @@ def update_flag(um, should_set, flag):
 
         update_flag(um, wildcard, UserMessage.flags.wildcard_mentioned)
 
+        is_me_message = getattr(message, 'is_me_message', False)
```

Comment:
I saw `is_me_message` flag is set during render_markdown.
and that's why it works before.

This line should help